### DISCUSSION TITLE
Loading Bundler .out projects with image list file in Pymeshlab

### DISCUSSION THREAD

**Discussion by JCAELA** [cnr-isti-vclab/PyMeshLab#183] [Q&A] [answered]
Hi,

I have been using Meshlab to texture meshes using Bundler .out projects exported from Colmap. In Meshlab, when I load the .out project (Using the Open Project button) it automatically asks for the image list file (.txt format). This way I have the mesh and the raster with cameras loaded in the project. After the surface reconstruction I simply use Texture>Parameterization+texture from registered cameras, and all goes well.

However, in Pymeshlab I am not able to load projects  in .out format. I am using the code:

import pymeshlab as ml
ms = ml.MeshSet()
ms.load_project('sparse_model.out')

Curiously, I get the following output: pymeshlab.pmeshlab.PyMeshLabException: The number of input files given (1) is different from the expected one (2).

I take it that the second file is the image list, but I dont know how to load it into the project.
Is there a workaround for this, or another way to texture models exported from Colmap?

Thanks in advance.

**Comment by alemuntoni** [comment, marked as answer]
Hi,
yes, actually this is not well documented, sorry about that. It is documented [here](https://pymeshlab.readthedocs.io/en/latest/classes/meshset.html#pmeshlab.MeshSet.load_project), but I agree that is not very intuitive.
When loading a bundler project, the `load_project` function can accept a list of strings, instead of a single string.
Therefore, something like:
```
ms.load_project(['sparse_model.out', 'img_list.txt'])
```
should work!
